plugin-server/src/main/jobs/local/mock-graphile-worker.ts

```diff
@@ -16,17 +14,25 @@ interface FsJob {
     pluginConfigId?: number
     pluginConfigTeam?: number
 }
-export class FsQueue extends JobQueueBase {
-    paused: boolean
-    started: boolean
+export class MockGraphileWorker {
```

Comment:
Just ignore this for now - it's a whole other refactor that needs to be done and I'm trying to keep this PR concise.

I want to do the minimal amount of work to make this "mock generic job queue work" but will likely later do away with this as it's not needed. Used in quite a few tests for now